packages/flutter_tools/README.md

```diff
@@ -1,13 +1,36 @@
 # Flutter Tools
 
-Developer tools for building Flutter applications.
+Command line developer tools for building Flutter applications.
 
-Be sure to follow the instructions on
-[CONTRIBUTING.md](../../CONTRIBUTING.md) to setup.
+## Working on Flutter Tools
 
-To run the tests, ensure that no devices are connected,
-then navigate to `flutter_tools` and execute:
+Be sure to follow the instructions on [CONTRIBUTING.md](../../CONTRIBUTING.md)
+to set up your development environment.
 
+### Running
+
+To run Flutter Tools from source, in this directory run:
+```shell
+$ ../../bin/cache/dart-sdk/bin/dart bin/flutter_tools.dart
```

Comment:
Add a note that this requires the dart_sdk to have been downloaded first from a `flutter` command.

Reply:
Added a "Setup" section